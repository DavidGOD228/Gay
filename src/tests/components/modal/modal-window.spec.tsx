import 'react-native'
// Note: test renderer must be required after react-native.
import React from 'react'
import renderer from 'react-test-renderer'
import ModalWindow from '../../../components/modal/modal-window'
import {
  Text
} from 'react-native'

it('renders correctly visible empty', () => {
  const tree = renderer.create(
    <ModalWindow visible={true}/>
  )
  expect(tree)
    .toMatchSnapshot()
})

it('renders correctly visible text', () => {
  const tree = renderer.create(
    <ModalWindow visible={true}>
      <Text>
        All you had to do was follow the damn train CJ
      </Text>
    </ModalWindow>
  )
  expect(tree)
    .toMatchSnapshot()
})
