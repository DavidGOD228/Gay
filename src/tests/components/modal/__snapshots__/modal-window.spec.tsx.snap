// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly visible empty 1`] = `
<Modal
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  transparent={true}
  visible={true}
>
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.90)",
        "flex": 1,
      }
    }
  />
</Modal>
`;

exports[`renders correctly visible text 1`] = `
<Modal
  hardwareAccelerated={false}
  onRequestClose={[Function]}
  transparent={true}
  visible={true}
>
  <View
    style={
      Object {
        "backgroundColor": "rgba(0,0,0,0.90)",
        "flex": 1,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
    >
      All you had to do was follow the damn train CJ
    </Text>
  </View>
</Modal>
`;
